htdocs/ecotaxdeee/sql/llx_ecotaxdeee.sql

```diff
@@ -0,0 +1,26 @@
+-- ===================================================================
+-- Copyright (C) 2002-2003 Rodolphe Quiedeville <rodolphe@quiedeville.org>
+-- Copyright (C) 2005-2009 Regis Houssin        <regis.houssin@inodbox.com>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <https://www.gnu.org/licenses/>.
+--
+-- ===================================================================
+
+CREATE TABLE IF NOT EXISTS llx_ecotaxdeee (
+                rowid INTEGER AUTO_INCREMENT PRIMARY KEY,
+                code VARCHAR(16),
+                amount DECIMAL(10,2),
+                date_creation DATETIME,
+                date_modification DATETIME NULL
```

Comment:
For this field, you must use
date_modification DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,